recipes/opendrift/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "opendrift" %}
-{% set version = "1.9.1rc1" %}
+{% set version = "1.9.1rc2" %}
```

Comment:
This falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
I have added the rc channel as specified in the CFEP.